tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '~> 0.9'
-  gem.add_dependency 'json'
-  gem.add_dependency 'simple_oauth'
-  gem.add_dependency 'oauth'
-  gem.add_dependency 'mime-types'
-  gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'rspec'
-  gem.add_development_dependency 'webmock'
-  gem.add_development_dependency 'simplecov'
+  gem.add_dependency 'json', '~> 1.8'
+  gem.add_dependency 'simple_oauth', '~> 0.3'
+  gem.add_dependency 'oauth', '~> 0.5'
+  gem.add_dependency 'mime-types', '~> 2.99'
```

Comment:
Same issue as with the json gem; 3.x doesn't work on 1.9.x. If you could offer any advice on how to fix that I would very much appreciate it. :)